README.md

```diff
@@ -174,6 +174,15 @@ Currently available `compiler/version` selections are:
 
 * `gnu/9.3.0` and `gnu/11.2.0`
 * `intel/2021.4.0`
+* `nvhpc/22.1` (use with `--with-gpu` on ac's GPU partition)
+
+*Please note: at the moment GPU-builds need to be done on the compute node.*
```

Comment:
This worked fine for me on login node... Is this still true, or am I getting this wrong somehow?

Reply:
Interesting... have you by any chance run the configuration phase before on a compute node already?

In any case, happy to drop the line as I expect this to work from login nodes as soon as access becomes more widely available.

Reply:
Nope, just tried again with a vanilla checkout:
```
[naml@ac6-100 naml]$ git clone git@github.com:ecmwf-ifs/dwarf-p-cloudsc.git dwarf-p-cloudsc-test
[naml@ac6-100 dwarf-p-cloudsc-test]$ git checkout nabr-hpc2020-nvhpc
[naml@ac6-100 dwarf-p-cloudsc-test]$ ./cloudsc-bundle create
[naml@ac6-100 dwarf-p-cloudsc-test]$ ./cloudsc-bundle build --clean --with-gpu --with-mpi --arch arch/ecmwf/hpc2020/nvhpc/22.1
...
[naml@ac6-100 dwarf-p-cloudsc-test]$ cd build && . env.sh
[naml@ac6-100 build]$ srun -N1 -q ng -p gpu --gres=gpu:4 --mem 200G --pty /bin/bash
[naml@ac6-300 build]$ bin/dwarf-cloudsc-gpu-scc-hoist 1 262144 128
```
And happy days, ~340GF/s. :grin: